web/lib/HabitatCommunity.js

```diff
@@ -257,27 +257,18 @@ button, .button {
             <p class='s'>RESERVE</p>
             <p id='totalReserve' class='xl light'> </p>
           </div>
-
-          <habitat-flip-card>
```

Comment:
No information tooltips anymore?

Reply:
@MaxKaay

Reply:
noticed it was left out here: https://www.figma.com/file/0ALyyzf2FiI4VAitpqUFrT/Habitat-(external)?node-id=2378%3A40 
I can easily migrate the flipcard code to HabitatCommunityPreview.js if desired